index.html

```diff
@@ -534,17 +532,21 @@ <h2>Issuer</h2>
         </dl>
         <pre class="example nohighlight" title="Usage of issuer properties">
 {
-  "id": "http://dmv.example.gov/credentials/3732",
-  "type": ["Credential", "ProofOfAgeCredential"],
-  <span class="highlight">"issuer": "https://dmv.example.gov/issuers/14"</span>,
-  <span class="highlight">"issued": "2010-01-01T19:73:24Z"</span>,
-  "claim": {
-    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-    "ageOver": 21
-  },
-  "proof": { ... }
-}
-        </pre>
+"verifiableCredential": {
+	"id": "< credential id >",
+	"type": ["ProofOfAgeCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01T19:73:24Z",
+	"claims": [{
+		"subject": {
```

Comment:
This completely breaks the data model.

Reply:
This does not break JSON nor my mental model of a verifiable credential. In fact it improves my mental model and understanding of a verifiable credential. 
I think you mean that this breaks some underlying hidden data model that is external to this specification but is not specified as being an integral part of it. This is part of the ambiguity that needs to be removed from the current specification.

Reply:
It breaks the consistency between the model and syntax when it is understood to be a graph. We can support both graph and tree based approaches -- so we should. There's no need to deviate.

Reply:
@dlongley. Can you please explain why the JSON structure I have proposed is not a graph. All it has done is added an extra node called 'claims'

Reply:
@David-Chadwick,

I did not say it didn't represent a graph. I said it breaks the existing consistency between the model and the syntax from the graph perspective -- and what we've done so far. If it is to be consistent with how everything else works, then it doesn't represent a graph we would prefer. So to accept this is to choose between consistency (and the oddity that results from it here) or inconsistency (the "breaking the data model" that @msporny speaks of).

To elaborate, the current graph model and syntax (JSON-LD) works like this:

```js
{
  "id": "<identifier of an RDF subject>",
  "<term referring to an RDF property>": {
    "id": "<identifier of another RDF subject in the object position>",
    /* so on */
  }
}
```

So something like this:

```js
{
  "id": "identifierA",
  "name": "Alice",
  "knows": {
    "id": "identifierB",
    "name": "Bob"
  }
}
```

Yields the triples:

```
<identifierA> <name> "Alice" .
<identifierB> <name> "Bob" .
<identifierA> <knows> <identifierB> .
```

This is inline with what's in the data model when talking about simple statements forming a graph.

When "id" is omitted, you can think of a graph node as merely existential, unlabeled, or "blank". These get assigned "blank node identifiers" (of the form `_:something`) in triple form. So, if we had this:

```js
{
  "id": "identifierA",
  "name": "Alice",
  "knows": {
    "name": "Bob"
  }
}
```

These are the triples:

```
<identifierA> <name> "Alice" .
_:b0 <name> "Bob" .
<identifierA> <knows> _:b0 .
```

This means Alice knows someone with the name Bob. We don't have an identifier for Bob, though.

Furthermore, we can embed graphs within graphs by defining the properties/terms we use as having values that occur in their own subgraphs. This is important for isolating groups of statements and what is done with "claim" presently. It allows each value of "claim" to be interpreted as a separate graph full of whatever triples are expressed therein.

So -- anyway, the problem with what you suggested is not with "claims" but with "subject". When you introduce "subject", if we are to be consistent with what we've done so far, then you create an entity that has no identifier that has a property "subject". I don't know what this relationship means or what this extra unidentified thing is. In other words, we have this:

A credential contains a claim which is a subgraph of statements. In that graph exists an unidentified thing that has a property "subject" that relates this unidentified thing (somehow) to another thing that we do identify.

What is this bizarre unidentified thing? Why does it exist? From a graph perspective, it has no purpose -- it appears to be added for reasons unrelated to consistency of the graph perspective or because of a misunderstanding of it. Why not eliminate it and just talk about the things you want to claim like usual?

Alternatively, we could define "subject" to have no real meaning in the graph data model -- causing it to be some kind of "pass through" nesting that only appears because we failed to model things properly. Either we make this explicit so graph-based processors can understand this blight, or we break consistency. Neither is a good choice, particularly when we're inventing something new, not trying to adapt something we're stuck with. It would be better not to break with how everything else works, by introducing a special case that causes additional cognitive load for those who understand and want to use the graph perspective. In fact, from a pure JSON perspective, I find the additional nesting of "subject" unappealing as well.

I think we can come up with something that doesn't cause this problem. Other standards groups have been quite successful at doing data modeling that works from both the JSON-LD graph perspective and the plain JSON tree perspective.